lib/neo4j/driver/auto_closable.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Neo4j
+  module Driver
+    # prepending methods with block execution
```

Comment:
Either document it properly or skip the comment. This does not help to understand. I plan to make it a separate gem where I would put more documentation, but it has to prove itself first here.